pytensor/tensor/extra_ops.py

```diff
@@ -751,82 +762,97 @@ def infer_shape(self, fgraph, node, ins_shapes):
                 out_shape = [pt_sum(repeats, dtype=dtype)]
         else:
             if repeats.ndim == 0:
-                out_shape[self.axis] = out_shape[self.axis] * repeats
+                out_shape[axis] = out_shape[axis] * repeats
             else:
-                out_shape[self.axis] = pt_sum(repeats, dtype=dtype)
+                out_shape[axis] = pt_sum(repeats, dtype=dtype)
         return [out_shape]
 
 
-def repeat(x, repeats, axis=None):
-    """Repeat elements of an array.
+def repeat(a: "TensorLike", repeats: TensorLike, axis: int or None) -> TensorVariable:
+    """Repeat elements of a tensor.
 
-    It returns an array which has the same shape as `x`, except along the given
-    `axis`. The `axis` parameter is used to specify the axis along which values
-    are repeated. By default, a flattened version of `x` is used.
+    See `numpy.repeat` for more information.
```

Comment:
What if I'm working in pure pytensor? I have to import numpy just to read our docs?

Reply:
you go to the docs :)

Reply:
We could add links like JAX does, I think we already have cross-ref so it should work the same way: https://github.com/jax-ml/jax/blob/e7acb20ea2ba037a0e82134a6b1bcf35430d28ca/jax/_src/numpy/lax_numpy.py#L6980

Then the docs have a clickable link: https://docs.jax.dev/en/latest/_autosummary/jax.numpy.repeat.html